### ISSUE TITLE
No response when upgrading the client

### ISSUE THREAD

**Issue by 99Kies** [informalsystems/hermes#3669]
https://github.com/cosmos/ibc-go/issues/4916

**Comment by 99Kies** [comment]
How should I do the upgrade?
```
➜  ~ hermes update client --height 11968505 --archive-address https://rpc.osmosis.zone:443 --restart-height 11968504 --host-chain vota-ash --client 07-tendermint-0
2023-10-20T02:34:51.207033Z  INFO ThreadId(01) using default configuration from '/home/ubuntu/.hermes/config.toml'
2023-10-20T02:34:51.207405Z  INFO ThreadId(01) running Hermes v1.6.0
ERROR foreign client error: error raised while updating client on chain vota-ash: failed building header with error: light client error for RPC address osmosis-1: rpc error: response error: Internal error: height 11958119 is not available, lowest height is 11969116 (code: -32603)
➜  ~ hermes update client --height 11969117 --archive-address https://rpc.osmosis.zone:443 --restart-height 11969116 --host-chain vota-ash --client 07-tendermint-0
2023-10-20T02:35:13.748345Z  INFO ThreadId(01) using default configuration from '/home/ubuntu/.hermes/config.toml'
2023-10-20T02:35:13.748709Z  INFO ThreadId(01) running Hermes v1.6.0
ERROR foreign client error: error raised while updating client on chain vota-ash: failed building header with error: light client error for RPC address osmosis-1: rpc error: response error: Internal error: height 11958119 is not available, lowest height is 11968504 (code: -32603)
➜  ~ hermes update client --height 11969117 --archive-address https://rpc.osmosis.zone:443 --restart-height 11969116 --host-chain vota-ash --client 07-tendermint-0
2023-10-20T02:35:18.997319Z  INFO ThreadId(01) using default configuration from '/home/ubuntu/.hermes/config.toml'
2023-10-20T02:35:18.997709Z  INFO ThreadId(01) running Hermes v1.6.0
ERROR foreign client error: error raised while updating client on chain vota-ash: failed building header with error: light client error for RPC address osmosis-1: rpc error: response error: Internal error: height 11958119 is not available, lowest height is 11968504 (code: -32603)
➜  ~ hermes update client --height 11969117 --archive-address https://rpc.osmosis.zone:443 --restart-height 11969116 --host-chain vota-ash --client 07-tendermint-0
2023-10-20T02:35:23.329287Z  INFO ThreadId(01) using default configuration from '/home/ubuntu/.hermes/config.toml'
2023-10-20T02:35:23.329630Z  INFO ThreadId(01) running Hermes v1.6.0
ERROR foreign client error: error raised while updating client on chain vota-ash: failed building header with error: light client error for RPC address osmosis-1: rpc error: response error: Internal error: height 11958119 is not available, lowest height is 11967286 (code: -32603)
➜  ~ hermes update client --height 11969117 --archive-address https://rpc.osmosis.zone:443 --restart-height 11969116 --host-chain vota-ash --client 07-tendermint-0
2023-10-20T02:35:26.815036Z  INFO ThreadId(01) using default configuration from '/home/ubuntu/.hermes/config.toml'
2023-10-20T02:35:26.815414Z  INFO ThreadId(01) running Hermes v1.6.0
ERROR foreign client error: error raised while updating client on chain vota-ash: failed building header with error: light client error for RPC address osmosis-1: rpc error: response error: Internal error: height 11958119 is not available, lowest height is 11969116 (code: -32603)
```

**Comment by 99Kies** [comment]
oh solved. this rpc didn't sync all block.
